gramex/data.py

```diff
@@ -1381,9 +1382,16 @@ def _type_conversion(param_list, operations=None):
 }
 
 
-def _filter_mongodb_col(col, op, vals, meta_cols):
+def _filter_mongodb_col(col, op, vals, meta_cols, object_key=None):
+    import bson
+
     if op in ['', '!']:
-        return {col: {_mongodb_op_map[op]: vals}}
+        convert = int if (meta_cols[col].dtype == pd.np.int64) else \
+            float if (meta_cols[col].dtype == pd.np.float64) else \
```

Comment:
Are we handling all relevant types:
- What about datetimes?
- Will pymongo ever return a type other than pd.np.float64 for float?
- Will pymongo ever return a type other than pd.np.int64 for int?
- What about booleans?